systemd/oracle-core.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Oracle Core v2.0
+Documentation=https://github.com/ergoplatform/oracle-core
+
+[Service]
+Type=simple
+# Config file is in ~/.config/oracle-core/
+ExecStart=bash -c "cd ~/.config/oracle-core; ~/.cargo/bin/oracle-core run"
```

Comment:
> Unfortunately systemd doesn't support relative paths in ExecStart/WorkingDirectory. Using a bash script is the best way I could find to do this.

Yes, but we're making more of a template than a ready-to-use systemd service file where operators will set their parameters. I just find `WorkingDirectory` a more explicit way of setting the working directory than cd'ing into it. I'm okay with both approaches.

> Hmm, you're right about the .config thing. I suppose a better way to do it would be to add the option to provide the path of oracle_config.yaml to oracle-core run

I agree we should pass a path to the config file. I made https://github.com/ergoplatform/oracle-core/issues/138

 
> edit: I suppose one way to do this, and also to automate the installation of systemd unit would be to use [cargo-xtask](https://github.com/matklad/cargo-xtask) to generate a unit file with the paths correctly set

It sounds interesting, but I'd rather keep it simple and wait for operator feedback. I suspect most of them will have pretty unique setups anyway.